superset/tasks/thumbnails.py

```diff
@@ -110,6 +111,8 @@ def cache_dashboard_thumbnail(
 # pylint: disable=too-many-arguments
 @celery_app.task(name="cache_dashboard_screenshot", soft_time_limit=300)
 def cache_dashboard_screenshot(
+    username: str,
+    guest_token: Optional[GuestToken],
     dashboard_id: int,
     dashboard_url: str,
     force: bool = True,
```

Comment:
Super nit, but what are your thoughts on moving `guest_token` down in the list and default it to `None`?:
``` python
def cache_dashboard_screenshot(
    username: str,
    dashboard_id: int,
    dashboard_url: str,
    force: bool = True,
    guest_token: Optional[GuestToken] = None,
    thumb_size: Optional[WindowSize] = None,
    window_size: Optional[WindowSize] = None,
) -> None:
```

Just so that all `Optional | None` arguments are in sequence.